examples/ex_cpu.c

```diff
@@ -0,0 +1,92 @@
+/* An example showing the use of al_get_cpu_count() and al_get_ram_size(). */
```

Comment:
I thought so too at first, but the log doesn't display or even give debug feedback on Android, so I coulnd't see anything when testing it. I made a different issue for that point, but it will probably be a while before someone implements support for native dialogs in Android... So for now, I have to use the display to see the results on Android. Don't mind if I keep it like that for now until the Android port is fixed?